contracts/src/DaypassPaymaster.sol

```diff
@@ -4,15 +4,20 @@ pragma solidity ^0.8.19;
 /* solhint-disable reason-string */
 
 import "../lib/account-abstraction/contracts/core/BasePaymaster.sol";
+import "../lib/account-abstraction/contracts/core/Helpers.sol";
 import "../lib/openzeppelin-contracts/contracts/token/ERC721/IERC721.sol";
 import "./Daypass.sol";
 
+error ExceedingGasLimit();
```

Comment:
I'm keeping these outside the contract so their more accessible from tests, it's how they have things in forge docs